flow/project.py

```diff
@@ -261,7 +261,7 @@ class JobOperation(object):
         :class:`dict`
     """
 
-    def __init__(self, id, name, job, cmd, directives=None):
+    def __init__(self, id, name, job, cmd, directives=None, run_job_operations=None):
```

Comment:
Is there a reason the default `JobOperation` needs a `run_job_operations` list?

Reply:
@bdice I don't think there's a need for this, is there? I think it's safe to remove.

Reply:
I think so. 🗑️